modules/apps/27-interchain-accounts/host/keeper/relay_test.go

```diff
@@ -410,7 +411,7 @@ func (suite *KeeperTestSuite) TestOnRecvPacket() {
 					Amount:      sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(100))),
 				}
 
-				data, err := icatypes.SerializeCosmosTx(suite.chainA.GetSimApp().AppCodec(), []sdk.Msg{msg})
+				data, err := icatypes.SerializeCosmosTx(suite.chainA.GetSimApp().AppCodec(), []proto.Message{msg})
```

Comment:
Should we add a test case expecting this to fail with a non `sdk.Msg` implementation? i.e. maybe include some msg response type as a `proto.Message`

Reply:
done!